book/invalidation.md

```diff
@@ -947,12 +945,17 @@ class BlockLayout:
     def layout(self):
         # ...
         if mode == "block":
-            # ...
+            if self.children.dirty:
+                # ...
```

Comment:
Also a random change to diffing that shouldn't be affected by this chapter but somehow is.